src/models/user.js

```diff
@@ -1,69 +1,52 @@
+import mongoose from 'mongoose';
 import bcrypt from 'bcrypt';
+import isEmail from 'validator/lib/isEmail';
+
+const UserSchema = new mongoose.Schema({
+  username: {
+    type: String,
+    unique: true,
+    required: true
+  },
+  email: {
+    type: String,
+    unique: true,
+    required: true,
+    validate: [isEmail, 'invalid email']
```

Comment:
'No valid email address provided.' instead of 'invalid email'

Reply:
fixed